fjord/base/tests/test_google_utils.py

```diff
@@ -33,8 +35,9 @@ def handler(*args, **kwargs):
 
 
 class TestGoogleUtils(TestCase):
-    @set_sys_argv(['test'])
-    def test_ga_track_event_test(self):
+    @override_settings(DEBUG=True)
+    @set_sys_argv([])
```

Comment:
It's essentially the same thing as overriding Django settings, but it's at a system level. Because it's at the system level, it could have other unintentional effects so it's less desirable.

Probably best to add a warning to the `set_sys_argv` docstring about that.

YOLO!